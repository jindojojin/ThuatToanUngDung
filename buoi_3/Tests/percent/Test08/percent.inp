90
3 10 10 11 66
12 0 10 0 78
6 0 5 22 67
3 10 5 22 60
0 0 15 0 85
3 0 10 0 87
6 0 5 22 67
9 10 15 0 66
3 0 5 11 81
6 10 15 0 69
9 10 10 0 71
9 0 5 11 75
9 0 15 22 54
6 10 5 11 68
1 0 0 0 99
3 10 10 0 77
9 0 0 22 69
12 0 10 0 78
0 10 10 22 58
3 10 5 22 60
12 10 0 11 67
3 0 10 22 65
3 0 0 0 97
3 10 5 11 71
9 0 5 22 64
9 10 15 0 66
12 0 5 22 61
9 0 5 22 64
12 0 10 0 78
6 10 5 0 79
3 0 5 0 92
0 10 15 11 64
3 0 10 11 76
3 0 10 11 76
6 10 15 0 69
6 10 5 0 79
0 10 5 22 63
3 0 10 11 76
3 0 5 22 70
0 10 10 22 58
0 10 0 22 68
9 10 15 0 66
6 0 15 22 57
12 0 0 0 88
9 10 0 11 70
3 0 5 11 81
9 0 0 22 69
6 10 0 11 73
9 10 15 0 66
0 10 5 11 74
12 0 15 0 73
0 10 15 22 53
9 10 0 11 70
12 10 5 22 51
0 10 5 0 85
9 10 10 22 49
9 0 10 0 81
0 0 10 22 68
12 0 5 11 72
6 0 10 11 73
9 10 10 22 49
9 10 15 11 55
3 10 10 0 77
6 10 0 22 62
6 0 15 22 57
12 0 0 0 88
3 0 10 0 87
0 2 48 48 2
6 10 15 22 47
0 0 15 11 74
3 0 5 0 92
0 10 10 11 69
6 10 0 11 73
6 10 0 22 62
12 0 5 22 61
3 10 0 0 87
0 10 10 11 69
3 0 5 0 92
9 10 15 0 66
12 0 15 11 62
12 10 10 11 57
9 10 5 11 65
3 10 0 11 76
0 0 10 11 79
3 10 10 22 55
9 10 5 11 65
6 0 0 22 72
9 0 5 11 75
3 0 5 11 81
0 26 24 50 0
