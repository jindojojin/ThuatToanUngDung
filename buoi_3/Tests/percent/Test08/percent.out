100
50
100
100
20
100
100
100
100
100
100
100
100
100
100
100
100
50
50
100
100
100
100
100
100
100
100
100
50
100
100
100
100
100
100
100
100
100
100
50
50
100
100
25
100
100
100
100
100
100
100
100
100
100
20
100
100
50
100
100
100
100
100
50
100
25
100
50
100
100
100
100
100
50
100
100
100
100
100
100
100
100
100
100
100
100
50
100
100
50
